### ISSUE TITLE
App Icons Not found?

### ISSUE THREAD

**Issue by NinjaJake86** [linuxserver/Heimdall#819]
Hi Guys,

Can't seem to use any icons other then the default one, even adding a supported app, it shows the icon in the menu to add it but once its added it can't find the icon on the home page, i've checked and the public/storage folder is empty, i've tryied to give it all the permissions i can so i don't think its an issue with permissions, any ideas?

Thanks!

**Comment by tomyvi** [comment]
There is currently an issue with SSL certificates validation at 2 moments : when checking if the application exists in the `SupportedApps.php` model and in the `ItemController.php` controller. 

In SupportedApps.php, line 134, you'll need to change to :
`$client = new Client(['http_errors' => false, 'timeout' => 60, 'connect_timeout' => 15, 'verify' => false]);`

In ItemsController.php, line 194, you'll need to change to :
```
$options=array(
    "ssl"=>array(
        "verify_peer"=>false,
        "verify_peer_name"=>false,
    ),
);  
$contents = file_get_contents($request->input('icon'), false, stream_context_create($options));
```

**Comment by KodeStar** [comment]
Should be fixed in 2.4.11
